java/README.md

```diff
@@ -9,11 +9,15 @@ client ID and client secret. You can create an ID/secret pair at:
 
 To build this code sample from the command line, type:
 
-  mvn compile
+  <code>mvn compile</code>
 
-To run the code sample from the command line, enter the following:
+To run a code sample from the command line, enter the following:
 
-  mvn exec:java -Dexec.mainClass="FULL_CLASS_NAME"
+  <code>mvn exec:java -Dexec.mainClass="FULL_CLASS_NAME"</code>
+
+For samples that require arguments, also specify -Dexec.args, e.g.:
+
+  <code>mvn exec:java -Dexec.mainClass="FULL_CLASS_NAME" -Dexec.args="arg1 arg2"</code>
```

Comment:
I tested these with maven. I had to install maven on my machine to test, and I guess I already had Java 1.7+. No objection to changing README to gradle. I would prefer it.